howto/setup-state-store/setup-azure-cosmosdb.md

```diff
@@ -63,6 +63,33 @@ spec:
     value: <REPLACE-WITH-COLLECTION>
 ```
 
+Here is an example of what it might look like:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+spec:
+  type: state.azure.cosmosdb
+  metadata:
+  - name: url
+    value: https://accountname.documents.azure.com:443
+  - name: masterKey
+    value: thekey==
+  - name: database
+    value: db1
+  - name: collection
+    value: c1
+```
+
+If you wish to use this as the actor secret store, append the following to the yaml.
+
+```yaml
+  - name: actorStateStore
+    value: "true"
+```
+
```

Comment:
What is the purpose of this new YAML example? The current document already outlines the plane YAML approach where the values include placeholder (e.g. `<REPLACE-WITH-URL>`) and a Kubernetes example which includes value from secret approach:

```yaml
secretKeyRef:
      name: <KUBERNETES-SECRET-NAME>
      key: <KUBERNETES-SECRET-KEY>
```

Reply:
In my opinion sometimes a template with "[abc]" but without actual values is unclear, and having real values (or values following the format of real values) helps clarify.  Not a strong opinion though, if this looks weird I'll remove.

Reply:
No, nothing weird, just I haven't seen actual values populates into docs for other bindings. It's possible I didn't look far enough. @orizohar thoughts?